background.ts

```diff
@@ -696,6 +709,12 @@ async function handleFailure(payload: any) {
 
 mainEvents.on('handle-failure', showErrorDialog);
 
+function doFullRestart() {
+  doK8sReset('fullRestart').catch((err: any) => {
+    console.log(`Error restarting: ${ err }`);
```

Comment:
Not sure what you mean by that.

Reply:
The only exception that can come out of `doK8sReset` is from the `handleFailure` function, which is called to deal with any exceptions that actually occurred during the restart. I only added the second exception handler because the IDE wanted it.